cmd/catalyst-node/catalyst-node.go

```diff
@@ -827,32 +827,35 @@ func (ac *PlaybackAccessControl) handleUserNew(payload []byte) []byte {
 	}
 
 	playbackID := lines[0]
-	token := requestURL.Query().Get("token")
+	jwtToken := requestURL.Query().Get("jwt")
 
 	var pubKey string
-	if token != "" {
-		claims, err := decodeJwt(token)
-
+	if jwtToken != "" {
+		claims, err := decodeJwt(jwtToken)
 		if err != nil {
 			glog.Errorf("Unable to decode JWT token %v", err)
 			return []byte("false")
 		}
 
-		if playbackID != claims.(jwt.MapClaims)["sub"].(string) {
+		if err := claims.Valid(); err != nil {
+			glog.Errorf("Invalid token %v", err)
+			return []byte("false")
+		}
```

Comment:
`decodeJwt` already calls this, so we don't need to check either. It will have returned an error checked on the previous `if`